catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/RefreshRegistryEntries.java

```diff
@@ -189,27 +230,44 @@ private RemoteRegistryResults getRemoteRegistryMetacardsMap() throws FederationA
         } catch (Exception e) {
             throw new FederationAdminException("Error querying for local metacards ", e);
         }
-        SourceResponse response;
+        List<Callable<RemoteResult>> tasks = new ArrayList<>();
```

Comment:
I was able to extract a couple of parts into private methods. Let me know if you think it is clearer this way.